lua/presence/init.lua

```diff
@@ -721,12 +710,12 @@ function Presence:get_buttons(buffer, parent_dirpath)
             "ssh",
         }
         local protocol, relative = repo_url:match("^(.+)://(.+)$")
-        if contains(protocols, protocol) == false or relative == nil then
+        if not vim.tbl.contains(protocols, protocol) or not relative then
```

Comment:
Oop, you're close but not quite 😅 The function name is `tbl_contains` (not a field on `vim.tbl` which doesn't exist):
> `Error executing vim.schedule lua callback: ...s/andrew/Code/presence.nvim/lua/presence/init.lua:7
13: attempt to index field 'tbl' (a nil value)`

Reply:
Oops